skimage/color/tests/test_colorlabel.py

```diff
@@ -8,37 +9,57 @@
                                      assert_array_equal, assert_warns)
 
 
+def test_deprecation_warning():
+
+    image = np.ones((3, 3))
+    label = np.ones((3, 3))
+
+    with pytest.warns(FutureWarning) as record:
+        label2rgb(image, label)
+
+    expected_msg = ("New recommanded value for bg_label is 0. "
+                    "Until version 0.19, default bg_label "
+                    "value is -1. Starting from version "
+                    "0.19, bg_label default value will be "
+                    "set to 0. To avoid this warning, please "
+                    "explicitly set bg_label value.")
```

Comment:
```suggestion
    expected_msg = "The new recommended value"
```

Reply:
I just wanted to display all the warning message, but short version is fine ;-)